http/jaxrs/src/test/java/io/quarkus/ts/security/core/SerializationIT.java

```diff
@@ -0,0 +1,51 @@
+package io.quarkus.ts.security.core;
+
+import static io.restassured.RestAssured.given;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.restassured.response.ResponseBodyExtractionOptions;
+
+@QuarkusScenario
+public class SerializationIT {
+
+    protected final static int LONGEST_EAGER_ALLOC = 100_000;
+
+    @Test
+    public void serializeString() {
+        String result = makeQuery("/hello/serialize-string").asString();
+        Assertions.assertEquals("Hello RESTEasy", result);
+    }
+
+    @Test
+    public void serializeList() {
+        String result = makeQuery("/hello/serialize-list").asString();
+        Assertions.assertEquals("[Hello RESTEasy]", result);
+    }
+
+    @Test
+    @Disabled("https://github.com/quarkusio/quarkus/issues/22654")
+    public void serializeBigList() {
+        String basePath = "/hello/big-serialize-list?expSize=";
+        mustBeGreaterThan(makeQuery(basePath + (LONGEST_EAGER_ALLOC - 39)).jsonPath().getInt("size"), 0);
```

Comment:
Too many magic numbers in the method. Probably, these should be separate constants

Reply:
These "magic numbers" are random variations just to tease out possible edge cases. 
The original scenario is the following one: https://github.com/FasterXML/jackson-databind/commit/3ccde7d938fea547e598fdefe9a82cff37fed5cb